cni_stop.sh

```diff
@@ -113,8 +113,26 @@ function unload_module {
 
 let status=0
 
+function load_inbox_drivers {
+    modprobe mlx5_core
+    modprobe ib_core
+}
+
+function delete_nic_operator_namespace {
+    if [[ !  -f "$WORKSPACE/mellanox-network-operator/deploy/operator-ns.yaml" ]];then
+        return 0
+    fi
+    nic_operator_namespace_file=$WORKSPACE/mellanox-network-operator/deploy/operator-ns.yaml
+    operator_namespace=$(grep -E '^[ ]+name:' $nic_operator_namespace_file | head -n 1 | cut -d: -f2 | tr -d ' ')
```

Comment:
you can install `yq`  tool to read/write values in yaml file

https://github.com/mikefarah/yq

perhaps open an improvement ticket for it.

Reply:
ok will switch to that. and open an issue to modify the rest of the project, since there are many places there where it would be better (more robust) to use the yq package.

Reply:
it seams we will not need to switch here, since this line will no longer be needed after deleting directly from the file. i opened a ticket anyway. for the other project.